src/extensions/extensionsApi.ts

```diff
@@ -149,7 +150,7 @@ export interface Param {
   label: string;
   description?: string;
   default?: string;
-  type?: ParamType;
+  type?: ParamType | SpecParamType;
```

Comment:
This is a bit strange, but we use Param all over the place, both in the context of a spec that we read from an extension.yaml file and one that was returned from the backend. In the first case this is SpecParamType, in the second this is ParamType.

Reply:
Ahhh, the actual issue here is that SpecParamType (the input the backend expects) is CamelCase, but ParamType (the output the backend returns) is ALLCAPS. This makes enum comparisons super awkward and buggy. I filed b/224618262 to fix this, and added a TODO.